docs/manual_installation.md

```diff
@@ -78,15 +80,15 @@ pip install -r requirements.txt
 ####
 - finaly start the bot
 ```bash
-./run.sh configs/config.json
```

Comment:
Why would you if the run.sh does it it self? that is only useful when you have it at a different location and that is not discussed in the manual installation.